views/blocks/thankyou.tpl

```diff
@@ -0,0 +1,9 @@
+[{if $oView->isModulePaymentMethod()}]
+    [{if $oView->isPaymentPending()}]
```

Comment:
I think we should use always check isPaymentPending on order object when deciding what to show to the user. Otherwise it could cause some inconsistency in text messages / sending emails / etc.

Reply:
needs to be clarified